src/rules/noUnusedVariableRule.ts

```diff
@@ -33,18 +39,58 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
     private skipParameterDeclaration: boolean;
     private skipVariableDeclaration: boolean;
 
+    // after a this.walk is completed, this.reactImport will be set to a NamespaceImport
```

Comment:
please move this comment block into `visitSourceFile`, where it is more relevant.